source/04-MX/02.ptx

```diff
@@ -90,105 +90,110 @@ and a group that means <m>T</m> is <em>bijective</em>.
     <statement>
         <p>
-  Let <m>T: \IR^3 \rightarrow \IR^3</m> be the linear transformation given by the standard matrix
-  <m>A=\left[\begin{array}{ccc} 2 &amp; -1 &amp; 0 \\ 2 &amp; 1 &amp; 4 \\ 1 &amp; 1 &amp; 3 \end{array}\right]</m>.
+Let <m>T: \IR^n \rightarrow \IR^n</m> be a linear bijection with standard matrix
+<m>A</m>.
         </p>
         <p>
-  Write an augmented matrix representing the system of equations given by <m>T(\vec x)=\vec{0}</m>,
-  that is, <m>A\vec x=\left[\begin{array}{c}0 \\ 0 \\ 0 \end{array}\right]</m>.
-  Then solve <m>T(\vec x)=\vec{0}</m> to find the kernel of <m>T</m>.
+By item (B) from <xref ref="MX2-card-sort"/>
+we may define an <term>inverse map</term><idx>inverse map</idx>
+<m>T^{-1} : \IR^n \rightarrow \IR^n</m> 
+that assigns <m>\vec x=T^{-1}(\vec b)</m> to be the unique vector such that
+<m>T(\vec x)=T(T^{-1}(\vec b))=\vec b</m>.
+        </p>
+        <p>
+Furthermore, let
+<me>A^{-1}=[T^{-1}(\vec e_1)\hspace{1em}\cdots\hspace{1em}T^{-1}(\vec e_n)]</me>
+be the standard matrix for <m>T^{-1}</m>. We call <m>A^{-1}</m> the
+<term>inverse matrix</term><idx>inverse matrix</idx> of <m>A</m>,
+and we also say that <m>A</m> is an <term>invertible</term><idx>invertible</idx>
+matrix.
         </p>
-    </statement>
-</activity>
-
-<sage language="octave">
-</sage>
-
-<definition>
-    <statement>
-    <p>
-  Let <m>T: \IR^n \rightarrow \IR^n</m> be a linear map with standard matrix <m>A</m>.
-    </p>
-<ul>
-<li><p>  If <m>T</m> is a bijection and <m>\vec b</m> is any <m>\IR^n</m> vector, 
-  then <m>T(\vec x)=A\vec x=\vec b</m> has a unique solution.
-</p></li>
-<li><p> So we may define an <term>inverse map</term><idx>inverse map</idx> <m>T^{-1} : \IR^n \rightarrow \IR^n</m> 
-  by setting <m>T^{-1}(\vec b)</m> to be this unique solution.
-</p></li>
-<li><p> Let <m>A^{-1}</m> be the standard matrix for <m>T^{-1}</m>. We call <m>A^{-1}</m> the
-<term>inverse matrix</term><idx>inverse matrix</idx> of <m>A</m>, so we also say that <m>A</m> is <term>invertible</term><idx>invertible</idx>.
-</p></li>
-</ul>
     </statement>
 </definition>
 
 
 <activity estimated-time='20'>
     <introduction>
         <p>
-  Let <m>T: \IR^3 \rightarrow \IR^3</m> be the linear transformation given by the standard matrix
-  <m>A=\left[\begin{array}{ccc} 2 &amp; -1 &amp; -6 \\ 2 &amp; 1 &amp; 3 \\ 1 &amp; 1 &amp; 4 \end{array}\right]</m>.
+Let <m>T: \IR^3 \rightarrow \IR^3</m> be the linear bijection given by the standard matrix
+<m>A=\left[\begin{array}{ccc} 2 &amp; -1 &amp; -6 \\ 2 &amp; 1 &amp; 3 \\ 1 &amp; 1 &amp; 4 \end{array}\right]</m>.
         </p>
     </introduction>
   <task>
       <p>
-  Write an augmented matrix representing the system of equations given by <m>T(\vec x)=\vec{e}_1</m>,
-  that is, <m>A\vec x=\left[\begin{array}{c}1 \\ 0 \\ 0 \end{array}\right]</m>.
-  Then solve <m>T(\vec x)=\vec{e}_1</m> to find <m>T^{-1}(\vec{e}_1)</m>.
+To find <m>\vec x = T^{-1}(\vec{e}_1)</m>, we need to find the unique
```

Comment:
To get students started on this problem, I've added an MCQ that helps them unravel $T(\vec x)=\vec e_1$ into the appropriate linear system.

Reply:
I like this.